server/server.go

```diff
@@ -577,6 +577,12 @@ func (s *Server) startReloadWatcher(ctx context.Context) {
 				logrus.Errorf("Unable to reload configuration: %v", err)
 				continue
 			}
+			// ImageServer compiles the list with regex for both
+			// pinned and sandbox/pause images, we need to update them
+			s.StorageImageServer().UpdatePinnedImagesList(
+				s.config.PauseImage,
```

Comment:
Totally non blocking nit: I think i'd prefer combining the config fields here instead of passing two args